CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+Version 1.12.0
+==============
+
+This release provides a number of exciting new features as well as a set
+of important bug fixes. Many thanks to our users for reporting these
+issues and @poplarShift @reckoner @jonmease for their code contributions
+and @scottclowe @syhooper for their contributions to the documentation.
+
+Features:
+
+- New plot options for controlling layouts including a responsive mode
+  as well as improved control over aspect using the newly updated bokeh
+  layout engine ([#3450](https://github.com/pyviz/holoviews/pull/#3450),
+  [#3575](https://github.com/pyviz/holoviews/pull/#3450))
+
```

Comment:
Should match formatting of previous entries which was without spaces between entries.

Reply:
Fixed in 107e313